reasoner/controller/ConcludableController.java

```diff
@@ -83,7 +84,6 @@ public void setUpUpstreamControllers() {
 
     @Override
     public void routeConnectionRequest(REQ req) {
-        if (isTerminated()) return;
```

Comment:
we no longer need manual check for whether the actor is terminated, messages are ignored if the actor is terminated at the higher level now